asv_bench/benchmarks/packers.py

```diff
@@ -547,6 +547,36 @@ def remove(self, f):
             pass
 
 
+class packers_write_json_lines(object):
+    goal_time = 0.2
+
+    def setup(self):
+        self.f = '__test__.msg'
+        self.N = 100000
+        self.C = 5
+        self.index = date_range('20000101', periods=self.N, freq='H')
```

Comment:
ok that's fine (though @joshowen make sure your example doesn't have dups as this is new code)